cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2789,6 +2790,7 @@ StatefulSet getKafkaStatefulSet()   {
         }
 
         @SuppressWarnings("deprecation")
+        @SuppressFBWarnings(value = "UPM_UNCALLED_PRIVATE_METHOD")
```

Comment:
Why is this needed and wasn't needed before?

Reply:
How was this resolved?